### PR TITLE
Improve docs for integer try convert

### PR DESCRIPTION
**Pull request by MatheusRich**
This started as fixing a typo (`to_ary` -> `to_int`), but I figured it would be good to have some examples to illustrate the usage. I didn't add an example for the exception case because other methods on the page also didn't have examples for exceptions.

### CODE REVIEW

File: numeric.rb [hunk 1]
```diff
@@ -233,11 +233,14 @@ def zero?
 #    Integer.try_convert(object) -> object, integer, or nil
 #
 #  If +object+ is an \Integer object, returns +object+.
+#    Integer.try_convert(1) # => 1
 #
 #  Otherwise if +object+ responds to <tt>:to_int</tt>,
 #  calls <tt>object.to_int</tt> and returns the result.
+#    Integer.try_convert("1") # => 1
```
Comment (nobu) [inline numeric.rb:240]: `String#to_int` is not defined by default.

**Review by nobu — CHANGES_REQUESTED** [review]
`to_int` example is wrong.

File: numeric.rb (hunk 1 above)
Comment (MatheusRich) [reply numeric.rb:240]: Do you think an example with `Float` would be better? Any other suggestions?

Comment (MatheusRich) [reply numeric.rb:240]:
```suggestion
#    Integer.try_convert(1.25) # => 1
```

**Review by nobu — APPROVED** [review]

**Comment by nobu** [comment]
Sorry, missed something.

**Pull request merged**